.github/workflows/ci_tests_dev.yaml

```diff
@@ -86,6 +86,14 @@ jobs:
             pcre
             zlib
 
+      # Install the GSHHG and DCW datasets on Linux/macOS to avoid corrupted files
+      # during GMT auto-downloading. Currently, we can't install them on Windows CI,
+      # because the Windows CI fails to compile "gshhg_version.c" due to unknown reasons
+      # (likely missing DLLs).
```

Comment:
Is there something we can do on https://github.com/conda-forge/gshhg-gmt-feedstock to get the `gshhg` conda-forge package to work with Windows?

Also, wanted to check if the compilation error only occurs with the `gshhg-gmt` package. I.e. can we still get `dcw-gmt` on Windows?

Reply:
> Is there something we can do on https://github.com/conda-forge/gshhg-gmt-feedstock to get the `gshhg` conda-forge package to work with Windows?

We're building the GMT dev source codes, so it is unrelated to the conda-forge package.

> Also, wanted to check if the compilation error only occurs with the `gshhg-gmt` package. I.e. can we still get `dcw-gmt` on Windows?

The gshhg_version program is used for checking the version information for both gshhg and dcw.